src/resources/attestations.js

```diff
@@ -58,16 +58,22 @@ class Attestations {
     }
   }
 
-  async phoneGenerateCode({ phone }) {
-    return await this.post('phone/generate-code', { phone })
+  async phoneGenerateCode({ country_calling_code, phone, method, locale }) {
```

Comment:
Could you document this interface change by submitting a PR in our [origin-docs repo](https://github.com/OriginProtocol/origin-docs) to update the [attestation APIs](https://github.com/OriginProtocol/origin-docs) ?

Same comment for the method phoneVerify below.

Thanks in advance !